cloudify_cli/env.py

```diff
@@ -505,7 +505,8 @@ def do_request(self, *args, **kwargs):
             copied_data = itertools.tee(kwargs.pop('data'),
                                         len(self._cluster))
 
-        kwargs.setdefault('timeout', self.default_timeout_sec)
```

Comment:
can't just use `setdefault` - timeout is None usually; but we do need the default timeout to be something short